pipeline/transformer/transformer_marc_common/src/main/scala/weco/pipeline/transformer/marc_common/transformers/MarcDataTransformer.scala

```diff
@@ -1,9 +1,18 @@
 package weco.pipeline.transformer.marc_common.transformers
 
+import weco.pipeline.transformer.marc_common.logging.LoggingContext
 import weco.pipeline.transformer.marc_common.models.MarcRecord
 
 trait MarcDataTransformer {
   type Output
 
   def apply(record: MarcRecord): Output
 }
+
+trait MarcDataTransformerWithLoggingContext {
```

Comment:
I think I will actually just add the implicit context argument onto MarcDataTransformer.apply, for consistency, but I wanted to keep the changes localised to the current task, so I've created a separate base trait for now.